README.md

```diff
@@ -79,20 +79,57 @@ vm.memory total=100
 vm.total_run_queue_lengths total=42,cpu=40,io=2
 ```
 
-On startup telegraf reporter will log something like:
+After you describe the telemetry supervision, you should append the aggregators config to your telegraf config.
+You can render aggregators config by calling
+
+```elixir
+config_string = TelemetryConfigTelegraf.Telegraf.ConfigAdviser.render(MyAppWeb.Telemetry.metrics(), options_kw)
+```
+
+you will get something like this:
 
 ```
-[info]  Suggested telegraf aggregator config for your metrics:
 [[aggregators.basicstats]]
 period = "30s"
 drop_original = true
 namepass = ["my_app.repo.query", "phoenix.endpoint.stop", "vm.memory", "vm.total_run_queue_lengths"]
 ```
 
-or you can render telegraf config manually by calling
+Copy/paste received config into your telegraf configuration file.
+If your final file looks like this, you're good to go.
 
-```elixir
-config_string = TelemetryConfigTelegraf.Telegraf.ConfigAdviser.render(MyAppWeb.Telemetry.metrics(), options_kw)
 ```
-
-Copy/paste aggregators config into your telegraf configuration file and you're good to go.
\ No newline at end of file
+  [agent]
```

Comment:
It'd be great if we can provide an example of a mix task with an embedded template like
```elixir
defmodule Mix.Tasks.MyApp.GenerateTelegrafConf do
  @moduledoc false
  use Mix.Task

  @config_path Path.join([Path.expand("../..", File.cwd!()), "config", "telegraf", "prod.conf"])
  @template ~S"""
   ...
  """
  def run(_) do
    aggregators_conf = TelemetryMetricsTelegraf.Telegraf.ConfigAdviser.render(MyApp.Telemetry.metrics(), [])
    :ok = File.write!(@config_path, @base_config <> aggregators_conf, [:write])
  end
end